java/test/jmri/jmrix/can/BundleTest.java

```diff
@@ -21,5 +21,23 @@
     public void testBadKey() {
             Bundle.getMessage("FFFFFTTTTTTT");
     }
+    @Test public void testGoodKeyMessageArg() {
```

Comment:
@pabender Is there an abstract test class that covers these methods that this BundleTest could inherit from?